tools/build_defs/framework.bzl

```diff
@@ -499,7 +497,7 @@ def _extract_link_params(cc_linking):
         cc_linking.dynamic_mode_params_for_executable,
     ]
 
-def _collect_libs(cc_linking):
+def collect_libs(cc_linking):
```

Comment:
Why are you making it not private?

Reply:
Nevermind, I see where it's used elsewhere

Reply:
Yep. I just split the commits.